### DISCUSSION TITLE
74lvc245 mandatory?

### DISCUSSION THREAD

**Discussion by emilen72** [markadev/AppleII-VGA#51] [General]
Hi, I have a maybe dummy question to ask.... I see other project on github that use pico or pico clone (purple) ad vintage computer/console without use of level-shifter and ask my self if is possible to eliminate the use of the 74lvc245 in this project.

Do you think is possible to make?

BR

Emiliano

https://github.com/aotta/PiCOLECO

https://github.com/robinhedwards/A8PicoCart

**Comment by markadev** [comment]
Hi there, for this VGA card design the 74LVC245 ICs have two purposes:
 * To multiplex the data & address lines into the Pico GPIOs
 * To provide 5V tolerant inputs for the Pico

They are a required part of this design for these two reasons.  It might be possible to substitute them for ones with a different technology as long as they can operate on 3.3V and are 5V tolerant (like a 74LS245 would not work).

**Reply by emilen72** [reply]
ok understood :-)

many thanks

**Reply by general-ackbar** [reply]
I tried to replace the 74LVC245 with 74HC245 and it seems to work. At least I haven't come across any problems yet. 74HC245 have higher power consumption and are slower, they can however operate on both 3.3V and 5V
